managers/users.js

```diff
@@ -89,7 +89,6 @@ function checkAndCreateUser(name, email, password, confirmation) {
                 .then(function (group) {
                   user.addGroups([group])
                   .then(function () {
-                    gitWorker.initNewGitRepo(name);
```

Comment:
This line is not required any more, since the repo will be automatically created at the first save request (allowed by `      auto_init: true` - libs/expressGit.js - line 13)